operator-common/src/main/java/io/strimzi/operator/common/operator/resource/StatusUtils.java

```diff
@@ -48,6 +50,25 @@ public static String iso8601(Instant instant) {
         return ZonedDateTime.ofInstant(instant, ZoneOffset.UTC).format(DateTimeFormatter.ISO_INSTANT);
     }
 
+    /**
+     *  Returns an Instant from a string date in ISO 8601 format
+     * @param date a string representing a date, for example "2019-07-23T09:08:12.356Z"
+     * @return an Instant
+     */
+    public static Instant iso8601(String date)  {
+        return Instant.parse(date);
+    }
+
+    /**
+     * Get an amount of time unit between a date and now
+     * @param date the date to start from
+     * @param unit the time unit of the interval, for example ChronoUnit.MINUTES
+     * @return long amount of time
+     */
+    public static long unitDifferenceUntilNow(Instant date,  ChronoUnit unit) {
```

Comment:
Since you only call it with `unit` as minutes we might as well push that into the method body and get rid of the parameter. Then we could call method this `numMinutesAgo`.